homework_04/05_viktoriya_vasileva/homework_04_05_viktoriya_vasileva.yaml

```diff
@@ -0,0 +1,34 @@
+name: Pylint
+
+on:
+    push:
+        branches: [ main ]
+        paths: 
+          - homework_04/05_viktoriya_vasileva/**
+    pull_request:
+        branches: [ main ]
+        paths: 
+          - homework_04/05_viktoriya_vasileva/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9"]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Analysing the code with pylint
+      run: |
+        cd homework_04
+        cd 05_viktoriya_vasileva
+        pylint circles.py
+        pylint test_circles.py
```

Comment:
After this you need another section for running the unit tests. For your case, this is:

- name: Run All Unit Tests
      run: |
        cd homework_04/05_viktoriya_vasileva
        pytest

I will just comment without accepting or returning the request, try to fix these problems, if you want help, ask.